pkg/lib/registry/registry.go

```diff
@@ -108,7 +109,7 @@ func (r RegistryUpdater) AddToRegistry(request AddToRegistryRequest) error {
 	return utilerrors.NewAggregate(errs) // nil if no errors
 }
 
-func populate(ctx context.Context, loader registry.Load, graphLoader registry.GraphLoader, querier registry.Query, reg image.Registry, refs []image.Reference, mode registry.Mode) error {
+func populate(ctx context.Context, loader registry.Load, graphLoader registry.GraphLoader, querier registry.Query, reg image.Registry, refs []image.Reference, mode registry.Mode, permissive bool, logger *logrus.Entry) error {
```

Comment:
nit: Seems like the logger being passed in is not used

Reply:
Sorry, yes, I originally logged a warning too.